apps/aechannel/src/aesc_offchain_state.erl

```diff
@@ -288,6 +289,10 @@ get_latest_signed_tx(#state{signed_tx = LastSignedTx})
     when LastSignedTx =/= ?NO_TX ->
     {tx_round(aetx_sign:tx(LastSignedTx)), LastSignedTx}.
 
+-spec get_latest_signed_trees(state()) -> aec_trees:trees().
```

Comment:
Why *signed* trees? Maybe `get_latest_trees/1`?
Cosmetical, so not a blocker

Reply:
I added "signed" to signify it corresponds to the "signed_tx". To be honest it's a little bit confusing. I will change it.